def testConnection(self):
    try:
        conn = self.connect()
        conn.ping()
        conn.close()
        return True
    except:
        raise CloudRuntimeException('Failed to Connect to DB')
